app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt

```diff
@@ -41,6 +42,15 @@ class MainActivity : AppCompatActivity() {
             }
     }
 
+    // Method to take the user to the profile page, rather than having to login every time
+    private fun profile(){
+        val profileButton = findViewById<ImageButton>(R.id.btnProfile)
+        profileButton.setOnClickListener {
+            val profileIntent = Intent(this, SavedSightingsActivity::class.java)
+            startActivity(profileIntent)
+        }
+    }
+
```

Comment:
This is the button that would take you directly to your profile. I have it taking directly to sightings because the login button currently does take you to the profile, it just makes you login first. If you do not want to implement this change, I suggest instead adding a "remember me" button so that the user does not have to login every time before viewing their profile page.

Reply:
If you use Firebase Auth, the default is to be remembered each time.  You don't need to force login.  So, an option is to have a user object, and only prompt for login if you notice the user object is null.